analysis/analyze-hotspots/build.gradle

```
@@ -36,6 +36,18 @@ dependencies {
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
 }
 
+task apiKey {
```

In the future this might be a bit more of a generic task, and mostly it creates the properties file, so if you choose to rename the variables inside, maybe we could call it:
```suggestion
task createProperties {
```

---

this is a really good point: and I might be tempted to break it up even more or give it an even more precise name.

So either:
1. One task to `createGradleProperties` and another to `writeApiKey` that depends on the `createGradleProperties` task;
2. Rename the task `createGradlePropertiesAndWriteApiKey`